Sources/ApiVideoPlayer/View/ApiVideoPlayer.swift

```diff
@@ -0,0 +1,39 @@
+#if !os(macOS)
+import SwiftUI
+
+@available(iOS 14, macOS 11.0, *)
+public struct ApiVideoPlayer: UIViewControllerRepresentable {
```

Comment:
`ApiVideoPlayer` is not explicit enough. It is disturbing because everything is this project could be called `ApiVideoPlayer`